implementations/rust/ockam/ockam_vault/src/storage/secrets_repository_sql.rs

```diff
@@ -50,15 +52,21 @@ impl SecretsRepository for SecretsSqlxDatabase {
             SigningSecretKeyHandle::ECDSASHA256CurveP256(_) => EC_DSA_SHA256_CURVE_P256.into(),
         };
 
-        let query = query("INSERT OR REPLACE INTO signing_secret VALUES (?, ?, ?)")
-            .bind(handle.to_sql())
-            .bind(secret_type.to_sql())
-            .bind(secret.to_sql());
+        let query = query(
```

Comment:
Can this affect AWS KMS vault? I'm asking because we don't have CI tests for it AFAIK, so could these changes potentially break something in a way that we don't catch it?

Reply:
If I'm not wrong this code is not called when an AWS KMS vault is used, is it?